common/src/main/scala/explore/AppMain.scala

```diff
@@ -64,11 +65,13 @@ trait AppMain extends IOApp {
     ReusabilityOverlay.overrideGloballyInDev()
 
     val initialModel = RootModel(
-      obsId =
-        Observation // TODO Remove this, it's here termporarily for testing URL automatic derivation.
-          .Id(ProgramId.Science.fromString.getOption("GS-2020A-DS-1").get, Index.One)
-          .some,
-      tabs = EnumZipper.of[AppTab]
+      tabs = EnumZipper.of[AppTab],
+      focused = // TODO Remove this, it's here termporarily for testing URL automatic derivation.
+        FocusedObs(
+          UUID.randomUUID
```

Comment:
Generally a randomUUID is considered a side effect. This is temporal so it should be ok

Reply:
That's good to know